tests/test_data_manager.py

```diff
@@ -152,6 +153,11 @@ def data_fs(fs):
     if EXT_PYDICOM:
         fs.add_real_directory(DATA_SRC, read_only=False)
 
+    # Add SSL certificate for PyPy
+    if platform.python_implementation() == "PyPy":
+        # Location for cert store on Ubuntu
+        fs.add_real_file("/etc/ssl/certs/ca-certificates.crt", read_only=True)
```

Comment:
Strange that this is only needed with PyPy... Yes, this is a bit inconvenient with pyfakefs.